### PR TITLE
Animate screenshot background while loading

### PR DESCRIPTION
**Pull request by cassidyjames**
Inspired by #1050, using the sort of shine placeholder animation that's become common on the web and in apps like Slack. If we think it's clear enough that it's loading, we could even remove the spinner altogether.

### CODE REVIEW

File: data/styles/loading.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        to right,
+        @bg_color 20%,
+        white,
+        @bg_color 80%
+    );
+}
+
+@keyframes loading {
+    from {
+        background-position: -600px center;
```
Comment (cassidyjames) [inline data/styles/loading.css]: I tried with percentages and it seems GTK didn't like that.

**Comment by cassidyjames** [comment]
Video of the effect: https://youtu.be/FNpZU_eaA10

File: data/styles/loading.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.loading {
+    animation: loading 1.5s ease-in infinite;
+    background-image: linear-gradient(
+        to right,
+        @bg_color 20%,
+        white,
```
Comment (danrabbit) [inline data/styles/loading.css]: For dark theme compatibility 
```suggestion
        @base_color,
```

Comment (cassidyjames) [reply data/styles/loading.css]: Ah you're smart. Thanks!

**Pull request merged**